packages/components/organisms/f-content-cards/src/components/templates/ContentCardVoucherCode.vue

```diff
@@ -0,0 +1,197 @@
+         */
+        voucherCopyCodeLabel () {
+            switch (this.copyState) {
+                case COPY_STATE_AVAILABLE:
+                    return this.copy.copyCodeLabel;
+                case COPY_STATE_TRANSITIONOUT:
+                    return this.copy.codeCopiedLabel;
+                default:
+                    return '';
+            }
+        },
+
+        /**
+         * Return the voucher copied label based on copy state
+         * @returns String
+         */
+        voucherCodeCopiedLabel () {
+            return this.copyState === COPY_STATE_COOLDOWN ? this.copy.codeCopiedLabel : '';
+        },
+
+        /**
+         * Return the voucher copy classes based on copy state
+         * @returns {*[]}
+         */
+        voucherCopyClasses () {
+            return [
+                this.$style['c-contentCard-voucher-copy'],
+                { [this.$style['c-contentCard-voucher-copy--copied']]: this.copyState === COPY_STATE_COOLDOWN },
+                { [this.$style['c-contentCard-voucher-copy--transition-out']]: this.copyState === COPY_STATE_TRANSITIONOUT }
+            ];
+        },
+
+        /**
+         * Checks to see if the copy state is in cool down
+         * @returns {boolean}
+         */
+        inCooldown () {
+            return this.copyState === COPY_STATE_COOLDOWN;
+        }
+    },
+    methods: {
+        /**
+         * Fires copy to clipboard, if in cool down clears the timeout, sets copy state
+         * creates a cool down timeout and emit voucher code click
+         */
+        copyVoucherCode () {
+            copyToClipboard(this.code);
+
+            if (this.copyState === COPY_STATE_COOLDOWN) {
+                clearTimeout(this.cooldownTimeout);
+            }
+
+            this.copyState = COPY_STATE_COOLDOWN;
+            this.cooldownTimeout = setTimeout(() => this.copyCooldownEnded(), 3000);
+            this.emitVoucherCodeClick(this.url);
+        },
+
+        /**
+         * Sets the copy state to transition timeout
+         */
+        copyCooldownEnded () {
+            this.copyState = COPY_STATE_TRANSITIONOUT;
+        },
+
+        /**
+         * Sets the copy state to state available
+         */
+        copyCooldownComplete () {
+            this.copyState = COPY_STATE_AVAILABLE;
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-contentCard-voucher {
+    display: flex;
+    width: 100%;
+    font-family: $font-family-base;
+    border: none;
+    cursor: pointer;
+    background: transparent;
+}
+
+.c-contentCard-voucher-code,
+.c-contentCard-voucher-copy {
+    @include font-size(body-l);
+    font-weight: $font-weight-bold;
+    color: $color-content-link;
+    width: 50%;
+}
+
+.c-contentCard-voucher-code {
+    text-align: left;
+}
+
+.c-contentCard-voucher-copy {
+    text-align: right;
+}
+
+.c-contentCard-voucher-copy--copied {
+    color: $color-content-default;
+}
+
+.c-contentCard-voucher-copy--transition-out {
```

Comment:
Apologies, but the dash in `transition-out` should be removed (and in the classes below) — switch them to camel-case and these all look good, e.g. `c-contentCard-voucher-copy--transitionOut`, `c-contentCard-voucher-copy--cooldownTickLeaveTo` etc